app/facades/foods_facade.rb

```diff
@@ -0,0 +1,14 @@
+class FoodsFacade
+
+  def food_search_results(query)
+    json = FoodsService.new.search_foods(query)
+    json[:foods].map do |food_data|
+      Food.new(food_data)
+    end.first(10)
```

Comment:
Good job limiting to 10 here, but I think we can be more efficient by limiting the data returned from the API to this limit - not always possible with every API service though. So, instead of mapping through every single result, why not limit `json[:foods].take(10)` first?

Reply:
love this. Thank you.